depends/packages/qt.mk

```diff
@@ -97,6 +97,7 @@ $(package)_config_opts_linux += -fontconfig
 $(package)_config_opts_linux += -no-opengl
 $(package)_config_opts_arm_linux  = -platform linux-g++ -xplatform $(host)
 $(package)_config_opts_i686_linux  = -xplatform linux-g++-32
+$(package)_config_opts_aarch64_linux = -xplatform linux-aarch64-gnu-g++
```

Comment:
`$(package)_config_opts_aarch64_linux = -xplatform linux-g++`

QT and other core dependencies compiled in an aarch64 docker container (multiarch/ubuntu-core) with the change above.  1.14-5-dev also compiled successfully.
https://hub.docker.com/r/multiarch/ubuntu-core

Reply:
Thanks!  I was able to build QT again using this.

I ran into this now upon configure:
`checking for static Qt plugins: -lqxcb -lxcb-static... no`
`configure: error: Could not resolve: -lqxcb -lxcb-static`